README.md

```diff
@@ -64,50 +71,52 @@ learn more about topics not directly related to a language to get this right! If
 I was doing Le Wagon today, I would read the resources below directly after it.
 
 1. Learn the most valuable Computer Science concepts in 1 afternoon with
-   [Computer Science Distilled](https://leanpub.com/computer-science-distilled).
-   Best book ever!
-2. Improve your Git level by reading
-   [Ry's Git Tutorial](https://www.amazon.com/Rys-Git-Tutorial-Ryan-Hodson-ebook/dp/B00QFIA5OC),
-   it is free and amazing!
-3. Enjoy a funny dive into DevOps with
-   [The Phoenix Project](https://www.amazon.com/Phoenix-Project-DevOps-Helping-Business/dp/0988262592)
+   [Computer Science Distilled]. Best book ever!
+1. Improve your Git level by reading
+   [Ry's Git Tutorial], it is free and amazing!
+1. Enjoy a funny dive into DevOps with [The Phoenix Project].
 
 ### Improve your Ruby skills
 
 If you know Ruby well, Rails will be so much easier!
 
-1. Read [Eloquent Ruby](http://eloquentruby.com/)
-2. Read [99 bottles of OOP](https://www.sandimetz.com/99bottles/)
+1. Read [Eloquent Ruby]
+1. Read [99 bottles of OOP]
+1. Do you Remember the "old days" when you were running Rubocop tests in your
+   terminal ? Do you run any still ? I'm sure you do. But let's [automate] it !
```

Comment:
Ahah, I love your article but honnestly I would not put it at the same level that the other 2 books ;)

Let's make "Great articles" section and add it there ;)